docs/source/release/v6.13.0/Framework/Bugfixes/39020.rst

```diff
@@ -0,0 +1 @@
+- ``getProtonCharge`` method of ``Run`` object was modified to account for period filtering on multi-period NeXus files.
```

Comment:
"Was modified" is a bit vague. Can we say what the new behaviour actually is? Something like "now returns a period filtered log rather than the total proton charge across all periods".